front/src/components/footer.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import styles from "~/styles/footer.module.scss"
+
+const Footer: React.FC = () => (
+  <footer>
+    <div className={styles["enshita-footer"]}>
+        <div className={styles["enshita-footer-content"]}>
+          <div className={`${styles["enshita-footer-logo"]} ${styles["enshita-footer-hidden"]}`}>
+            enshita
+          </div>
+          <div className={styles["enshita-footer-menu"]}>
+            <div className={styles["enshita-footer-about"]}>About Growthers</div>
```

Comment:
リストになっているので無理にgridで合わせるんじゃなくて箇条書きの部分はlistにしたほうが良いでしょう

```suggestion
            <div className={styles["enshita-footer-about"]}>About Growthers</div>
            <ul>
              <li>
```

Reply:
[036a01e](https://github.com/Growthers/enshita/pull/49/commits/036a01e366f158cfd3d9d53515c60988e7e44f25)
`ul` `li` tagを用いました。それに伴いgridをやめて最小限の`flex`のみで対応しました